controllers/apps.js

```diff
@@ -45,10 +46,61 @@ exports.getApps = asyncWrapper(async (req, res, next) => {
   const useOrdering = await Config.findOne({
     where: { key: 'useOrdering' }
   });
+  const useDockerApi = await Config.findOne({
+    where: { key: 'dockerApps' }
+  });
+  const unpinStoppedApps = await Config.findOne({
+    where: { key: 'unpinStoppedApps' }
+  });
 
   const orderType = useOrdering ? useOrdering.value : 'createdAt';
   let apps;
 
+
+  if (useDockerApi && useDockerApi.value==1) {
+    let {data:containers} = await axios.get('http://localhost/containers/json?{"status":["running"]}', {
+      socketPath: '/var/run/docker.sock'
+    });
+
+    if (containers) {
+      apps = await App.findAll({
+        order: [[ orderType, 'ASC' ]]
+      });
+
+      containers = containers.filter((e) => Object.keys(e.Labels).length !== 0);
+      const dockerApps = [];
+      for (const container of containers) {
+        const labels = container.Labels;
+
+        if ('flame.name' in labels && 'flame.url' in labels && (labels['flame.type']==='application' || labels['flame.type']==='app')) {
```

Comment:
Checking for `app` or `application` can be shortened to `/^app/.test(labels['flame.type'])`

Reply:
I made it that way because in the future there can be a bookmark option, but until that time, is better the way you pointed.